functions/EmuScripts/emuDeckCitra.sh

```diff
@@ -102,7 +103,31 @@ Citra_setABXYstyle(){
 
 #Migrate
 Citra_migrate(){
-echo "NYI"
+echo "Begin Citra Migration"
+    emu="Citra"
+    migrationFlag="$HOME/.config/EmuDeck/.${emu}MigrationCompleted"
+    #check if we have a nomigrateflag for $emu
+    if [ ! -f "$migrationFlag" ]; then
+        #citra flatpak to appimage
+        #From -- > to
+        migrationTable=()
+        migrationTable+=("$HOME/.var/app/org.citra_emu.citra/data/citra-emu" "$HOME/.local/share/citra-emu")
+        migrationTable+=("$HOME/.var/app/org.citra_emu.citra/config/citra-emu" "$HOME/.config/citra-emu")
+
+        # migrateAndLinkConfig "$emu" "$migrationTable"
+    fi
+
+    #move data from hidden folders out to these folders in case the user already put stuff here.
+    origPath="$HOME/.var/app/"
```

Comment:
I think this is not correct - it would be a bit longer.

"$HOME/.var/app/org.citra_emu.citra/data/citra-emu" ( i think)

Reply:
That is the correct path, yes.